typings/third-party-defs.d.ts

```diff
@@ -3,5 +3,6 @@ declare var MIDI: any;
 declare var Guppy: any;
 declare var Sk: any;
 declare var safari: any;
```

Comment:
Drop safari and system (see earlier comment in #6441)

Reply:
Done.